common/path/platform/c++17/path.cpp

```diff
@@ -46,6 +46,16 @@ bool FileExists(const string &path) {
 	return fs::exists(path);
 }
 
+bool FileDelete(const string &path) {
+	error_code ec;
+	bool deleted = fs::remove(fs::path(path), ec);
+	if (not deleted) {
+		log::Error("Failed to remove the file: " + path + " error: " + ec.message());
+		return false;
```

Comment:
Maybe it's better if this function returns an error. This is a quite generic function, and logging is not always what you want (it could be expected to fail, for example). The caller can log instead.

Reply:
Agreed, fixing